package.json

```diff
@@ -87,7 +88,17 @@
     "gh-pages": "0.12.0",
     "html-webpack-plugin": "2.28.0",
     "imports-loader": "0.7.1",
+    "istanbul-instrumenter-loader": "^2.0.0",
```

Comment:
@lexzhukov Well, I haven't prior experience with yarn. Launched `yarn` at the root, `yarn.lock` got updated resulting in a lots of changes in the file. If I did something wrong, please let me know.